server/api/ping_handler.py

```diff
@@ -1,14 +1,12 @@
 import json
 from flask import jsonify, request, Blueprint
 from config import Config
-ping_handler = Blueprint('ping_handler', __name__)
+from payments.payout_project import payout_project
 
+ping_handler = Blueprint('ping_handler', __name__)
 
-@ping_handler.route('/api/v1/ping', methods=['POST'])
+@ping_handler.route('/api/v1/ping', methods=['GET'])
 def ping():
-    if request.method == 'POST':
-        body = json.loads(request.get_data())
-        if body['teamName'] in Config.TEAM_NAME.split(', '):
-            return jsonify({'response': "{} is now part of the team".format(body['teamName'])}), 200
-        else:
-            return jsonify({'response': "{} is not part of the team, change your .env".format(body['teamName'])}), 400
+    funds = payout_project(3)
```

Comment:
is this just hardcoded demo project_id `3` for now?

Reply:
oh wait yeah this is under /ping route nvm

Reply:
Yes, sorry I should have made this more clear. I was using the ping route to test outputs